bytecomp/translmod.ml

```diff
@@ -656,8 +641,30 @@ and transl_structure loc fields cc rootpath final_env = function
                transl_module Tcoerce_none None modl, body),
           size
 
+      | Tstr_open od ->
+          begin match od.open_expr.mod_desc with
+          | Tmod_ident _ ->
```

Comment:
This case is not actually needed. The optimised output will be produced anyway: `open_type` is empty and the binding of `mid` will be marked `Alias` and removed by `Simplif`.